libs/expo/betterangels/src/lib/screens/Clients/index.tsx

```diff
@@ -230,6 +230,7 @@ export default function Clients({ Logo }: { Logo: ElementType }) {
           renderItem={({ item: clientProfile }) =>
             clients ? (
               <ClientCard
+                imageUrl={clientProfile.profilePhoto?.url}
```

Comment:
I think this also needs to be done in `NoteCard`

Reply:
don't see on NoteType under client a profile photo. we can look at it together